nav2_costmap_2d/include/nav2_costmap_2d/denoise/image.hpp

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) 2022 Andrey Ryzhikov
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NAV2_COSTMAP_2D__IMAGE_HPP_
+#define NAV2_COSTMAP_2D__IMAGE_HPP_
```

Comment:
According to naming rules, the macro should be `NAV2_COSTMAP_2D__DENOISE__IMAGE_HPP_`

Reply:
Fixed 9830245012b72fcb316d8076fba1f08d7088e734